Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash

```diff
@@ -0,0 +1,260 @@
+record Wishable {
+    // If the wishable is an item, set that here. Otherwise, use $item[none].
+    item theItem;
+
+    // If the wishable is an effect, set that here. Otherwise, use $effect[none].
+    effect theEffect;
```

Comment:
Wish I could union theItem and theEffect together but ASH doesn't support this AFAICT.